api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -81,6 +83,11 @@ private class PurchasesAPI {
         purchases.purchasePackage(activity, packageToPurchase, upgradeInfo, purchaseChangeCallback)
         purchases.purchasePackage(activity, packageToPurchase, purchaseCallback)
 
+        purchases.purchaseProductWithOption(activity, storeProduct, purchaseOption, upgradeInfo, purchaseChangeCallback)
```

Comment:
hmmm it feels a little cumbersome from the developer's perspective to pass the storeproduct AND the purchaseoption, since from their point of view the purchaseoption is directly linked with a single storeproduct... 

i know we need info from both though 🤔 i suppose it'd be overkill to cache storeProducts and be able to look up the storeproduct associated with the purchase option that is passed in? idk just thinking out loud

Reply:
Well, maybe a little. The option you suggested sounds interesting but seems like adding extra complexity for a small benefit... Personally I think passing the product and the option is not a big deal, but happy to hear other people's thoughts.

Reply:
yeah i'm okay with that. we can iterate on the API later if we need to. one other thought would be to add the product details and any other info we need to the purchaseoption, so that we can use only that to purchase. but definitely not necessary to think about this more for this PR